internal/servers/controller/interceptor.go

```diff
@@ -230,9 +230,7 @@ func isNil(i interface{}) bool {
 	return false
 }
 
-func auditRequestInterceptor(
-	_ context.Context,
-) grpc.UnaryServerInterceptor {
+func auditRequestInterceptor() grpc.UnaryServerInterceptor {
```

Comment:
I would have these still take `_ context.Context` so they are consistent with the other interceptor functions

Reply:
Addressed in [1063c66](https://github.com/hashicorp/boundary/pull/1874/commits/1063c6668fd1da5c1484d248c8d0cefead6a9078)